src/components/Nav/MobileMenu.jsx

```diff
@@ -32,7 +32,7 @@ export const MobileMenu = {
     borderRadius: "5px",
   },
   bmMenuWrap: {
-    position: "fixed",
```

Comment:
Won't this cause issues when scrolling?

Reply:
yes it will, good catch, i didn't mean to push this, was just experimenting.